gradle-baseline-java-config/resources/checkstyle/checkstyle.xml

```diff
@@ -98,8 +105,11 @@
             <property name="setterCanReturnItsClass" value="true"/>
         </module>
         <module name="HideUtilityClassConstructor"/> <!-- Java Coding Guidelines: Private constructors -->
+        <module name="IllegalCatch">
```

Comment:
I believe we had this originally, but then removed it because people found it too constraining.